consoles/VNC.pm

```diff
@@ -159,7 +164,7 @@ sub login {
 }
 
 sub _handshake_protocol_version {
-    my $self = shift;
+    my ($self) = @_;
```

Comment:
I never said anything else. I just once told you to the reason why others prefer the shift